src/System.Net.Primitives/tests/FunctionalTests/IPAddressParsing.cs

```diff
@@ -286,6 +286,8 @@ public void ParseIPv4_InvalidAddress_ThrowsFormatExceptionWithInnerException(str
             new object[] { "1::%1", "1::%1" },
             new object[] { "::1%12", "::1%12" },
             new object[] { "::%123", "::%123" },
+            new object[] { "Fe08::1%13542", "fe08::1%awdl0" },
+            new object[] { "Fe08::1%13542", "fe08::1%gpd0" },
```

Comment:
Do we have tests for valid textual scope IDs that don't actually map to anything?

Reply:
Well these are (I don't know what interfaces CI have), added one more just in case 🙂